go/tools/builders/generate_checker_main.go

```diff
@@ -31,15 +31,206 @@ import (
 var codeTpl = `
 package main
 
+import (
```

Comment:
As you mentioned in the PR comment, this has gotten pretty large, and it seems like the only parts that need to be generated are the blank imports of the checker libraries.

I think it's probably cleaner to move this code to a new file, perhaps `//go/tools/builders:checker_main.go`. You can define a `go_source` rule in `//go/tools/builders` that includes this code, as well as flags.go (so you don't need to redefine `multiFlag`). In the `go_checker` rule, you can list that target in `embed` in the `attr` struct passed to `library_to_source`. The generated file would then just contain the blank imports.